File: frontend/src/types/purchaseRecommendation.ts
export type RequiredOrderItem = {
  quotation_item_id: string
  quotation_document_id: string
  quotation_no: string
  item_code: string
  item_name: string
  customer_name: string | null
  unit_price: number | null
  delivery_deadline: string | null
}

File: backend/app/schemas/purchase_recommendation.py
from pydantic import BaseModel


class RequiredOrderItemResponse(BaseModel):
    quotation_item_id: str
    quotation_document_id: str
    quotation_no: str
    item_code: str
    item_name: str
    customer_name: str | None = None
    unit_price: float | None = None
    delivery_deadline: str | None = None
